cross-project-tests/debuginfo-tests/dexter/dex/debugger/visualstudio/VisualStudio.py

```diff
@@ -307,6 +307,30 @@ def set_current_stack_frame(self, idx: int = 0):
                 )
             )
 
+    def _translate_stop_reason(self, reason):
+        # https://learn.microsoft.com/en-us/dotnet/api/envdte.dbgeventreason?view=visualstudiosdk-2022
+        if reason == 1:  # dbgEventReasonNone
```

Comment:
Can those magic values (1, 9, 8, ...) be replace for some named constants?

Reply:
May be something like:
```
dbgEventReasonNone = 1
dbgEventReasonGo = 2
...
if reason == dbgEventReasonNone: 
...
```
Basically is the same code, but I think it is easy to read.

Reply:
I don't have strong opinions, but in general even if it's only going to be used once I think it's probably better to have something like `class VSDebugEventReason(IntEnum):` with the correct values added to it.